extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -259,14 +260,66 @@ export class MigrationCutoverDialog {
 					fontSize: '12px'
 				}).component();
 
+				const _emptyTableImage = view.modelBuilder.image().withProps({
+
+					iconPath: IconPathHelper.emptyTable,
+					iconHeight: '100px',
+					iconWidth: '100px',
+					height: '100px',
+					width: '100px',
+					CSSStyles:{
+						'text-align':'center'
+					}
+
+				}).component();
+
+				const _emptyTableText = view.modelBuilder.text().withProps({
+					value: "No backup files to show",
+					CSSStyles:{
+						'text-align':'center',
+						'font-size':'large',
+						'font-weight':'bold',
+						'width':'300px'
+					}
+				}).component();
+
+				const _emptyTableSubText = view.modelBuilder.text().withProps({
+					value: "If you expected to see results, please check the connection to the sql server instance",
```

Comment:
1. As the hygiene check indicates you should use single quotes for strings, not double quotes
2. This should be localized - see other examples in this file (`loc.<var>`)

Same for the other display strings in this file

Reply:
Thanks, will fix it.